apps/i18n/maze/fr_fr.json

```diff
@@ -8,8 +8,8 @@
   "collectorCollectTooltip": "Collecte un objet",
   "collectorCollectedEverything": "Bravo! Tu as collecté {count, plural, one {the item} other {all # items}}!",
   "collectorCollectedNothing": "Continue le codage! Tu dois collecter autant d'objets que tu peux.",
-  "collectorCollectedSome": "Beau boulot! Tu as collecté {count, plural, one {1 objet} other {# objets}}!",
-  "collectorCollectedNotEnough": "Continuez à coder! Vous devez collecter au moins {goal, plural, one {1 objet} other {# objets}}.",
+  "collectorCollectedSome": "C'est du bon travail ! Tu as collecté {count, plural, one {1 objet} autre(s) {# objets}}!",
+  "collectorCollectedNotEnough": "Continue à coder ! Tu dois collecter au moins {goal, plural, one {1 objet} autre(s) {# objets}}.",
```

Comment:
@castro-jorge got a couple errors here where the translator translated MessageFormat syntax